scripts/output/single/reporting.R

```diff
@@ -76,7 +76,7 @@ edgetOutputDir <- file.path(outputdir, "EDGE-T")
 if(file.exists(edgetOutputDir)) {
   if (! file.exists(file.path(edgetOutputDir, "4_Output", "vehSalesAndModeShares.RDS"))) {
     message("EDGE-T reporting files are missing, probably because the run was killed.")
-    message("Rerunning toolIterativeEDGETransport().")
+    message("Rerunning iterativeEdgeTransport().")
```

Comment:
is this change on purpose?

Reply:
it was a merge overwrite that did not affect the runs. I reverted it back to avoid the change.